enhancements/installer/aws-custom-edge-machineset-local-zones.md

```diff
@@ -0,0 +1,853 @@
+- the VPC must have enough free CIDR blocks to create subnets in Local Zones
+- the Local Zones group must be activated manually
+- the subnets must be created manually
+- the subnet must be associated with a route table
+- the instance type offers must be checked, and the type is chosen for each Local Zone
+- the MachineSets manifests must be created manually for each zone Local
+- preferably, the node should not receive regular cluster workloads, mainly ingress. due the
+  unsupported network load balancers, latency, and costs.
+
+Presently, the installer does not support Local Zones subnets on day zero operations
+for those reasons:
+- the Local Zones do not support network load balancers (NLB or CLB), used by
+  ingress deployed on the `worker` pool.
+- the Local Zones do not support Nat Gateway, used as a gateway on the default IPI
+  deployment for each subnet and availability zone.
+- the default instance type for compute nodes used on IPI, `m6i.xlarge`, is not
+  available on Local Zones
+- the default EBS type for IPI, `gp3`, is not available in the most Local Zones locations
+- segment the `worker` compute node pool is not allowed
+
+
+### User Stories
+
+Phase 1:
+
+- As a cluster admin, I want the ability to specify a set of subnets on the AWS
+  Local Zones locations to deploy worker nodes, so I can create custom applications
+  to deliver low latency to my end users.
+
+- As a cluster admin, I would like to create a cluster extending worker nodes to the
+  edge of the AWS cloud provider with Local Zones, so I can create custom applications
+  to deliver low latency to my end users.
+
+- As a cluster admin, I would like to control the workloads running on the edge
+  machine pool, and keep it dedicated to applications that are latency sensitive,
+  so I can have a cost-effective infrastructure, avoiding mixing workloads in different locations.
+
+- As a cluster admin, I would like to select existing subnets from the local and
+  the parent region zones, to install a cluster, so I can manage my custom networks with
+  my automation.
+
+Phase 2:
+
+- As a cluster admin, I would like to install OpenShift clusters, extending the
+  compute nodes to the Local Zones in my day-zero operations without needing to
+  set up the network and compute dependencies, so I can speed up the edge adoption
+  in my organization using OKD/OCP.
+
+### Goals
+
+Phase 1:
+
+- Admins can create machine pools named `edge` when installing a cluster in existing VPC with subnets in AWS Local Zones.
+- Admins can select and configure subnets on Local Zones before cluster creation.
+- Ensure the installer allows creating a new machine pool for `edge` workloads
+- Ensure the installer can create the MachineSet with `NoSchedule` taints on edge machine pools.
+- Ensure Local Zones subnets will not be used on `worker` machine pools or control planes.
+- Ensure the Wavelength zone will not be used in any machine pool
+
+Phase 2:
```

Comment:
In Phase 2, the installer is creating network resources... that does not include the application load balancers, right? The ALBs will always be created Day 2?

Reply:
`Network` I would mean every manual resource created by the user on Phase-1 to install a cluster in the existing network/VPC: VPC, subnets (regular and LZ), and dependencies.

Yes, ALB should be installed on Day 2, the user can opt to use it or not, depending on the use case they've chosen to use Local Zones.

I will clarify the second bullet to keep it more explicitly of resources Phase 2 will create. Thanks for pointing it out.

Reply:
@patrickdillon I described which network resources on this section. Thanks